grabbit/src/main/groovy/com/twcable/grabbit/server/batch/ServerBatchJob.groovy

```diff
@@ -89,16 +90,19 @@ class ServerBatchJob {
     static class PathBuilder {
         final ConfigurationBuilder configurationBuilder
         String path
+        String excludePaths
 
 
         PathBuilder(ConfigurationBuilder configurationBuilder) {
             this.configurationBuilder = configurationBuilder
         }
 
-
-        ContentAfterDateBuilder andPath(String path) {
-            if (path == null) throw new IllegalArgumentException("path == null")
+        ContentAfterDateBuilder andPath(String path, String excludePaths) {
+            if(path == null) throw new IllegalArgumentException("path == null")
             this.path = path
+            if(excludePaths != null && !excludePaths.isEmpty()) {
+                this.excludePaths = excludePaths
+            }
```

Comment:
Of course this line becomes just a little different with `Collection<String>`...